src/interned.rs

```diff
@@ -253,6 +347,12 @@ where
         self.data(db, C::deref_struct(s))
     }
 
+    pub fn reset(&mut self, db: &mut dyn Database) {
+        // Trigger a new revision.
+        let _zalsa_mut = db.zalsa_mut();
```

Comment:
```suggestion
        // Trigger a new revision.
        db.zalsa_mut().new_revision();
```
The behavior here has changed on master.